plothist/comparison.py

```diff
@@ -164,16 +164,15 @@ def get_pull(h1, h2, h1_uncertainty_type="symmetrical"):
         uncertainties_low, uncertainties_high = get_asymmetrical_uncertainties(h1)
         h1_variances = np.where(
             h1.values() >= h2.values(),
-            uncertainties_low ** 2,
-            uncertainties_high ** 2,
+            uncertainties_low**2,
```

Comment:
Is this fixed now? every time when one of us applied black this modification happened.

Reply:
I tried to apply the pre-commit hooks, I get the same result.